.bandit

```diff
@@ -0,0 +1,399 @@
+
```

Comment:
You should exclude it in `.gitattributes` so we wouldn't pack the file.

Reply:
Updated